packages/@okta/vuepress-site/conductor.yml

```diff
@@ -2793,3 +2793,51 @@ redirects:
+    to: /docs/guides/sign-into-web-app/-/create-okta-application/
+  - from: /guides/sign-into-web-app/-/define-callback/index.html
+    to: /docs/guides/sign-into-web-app/-/define-callback/
+  - from: /guides/sign-into-web-app/-/get-user-info/index.html
+    to: /docs/guides/sign-into-web-app/-/get-user-info/
+  - from: /guides/sign-into-web-app/-/next-steps/index.html
+    to: /docs/guides/sign-into-web-app/-/next-steps/
+  - from: /guides/sign-into-web-app/-/redirect-to-sign-in/index.html
+    to: /docs/guides/sign-into-web-app/-/redirect-to-sign-in/
+  - from: /guides/sign-into-web-app/-/require-authentication/index.html
+    to: /docs/guides/sign-into-web-app/-/require-authentication/
+  - from: /guides/sign-into-web-app/aspnet/before-you-begin/index.html
+    to: /docs/guides/sign-into-web-app/aspnet/before-you-begin/
+  - from: /guides/sign-into-web-app/aspnet/configure-packages/index.html
+    to: /docs/guides/sign-into-web-app/aspnet/configure-packages/
+  - from: /guides/sign-into-web-app/aspnet/create-okta-application/index.html
+    to: /docs/guides/sign-into-web-app/aspnet/create-okta-application/
+  - from: /guides/sign-into-web-app/aspnet/define-callback/index.html
+    to: /docs/guides/sign-into-web-app/aspnet/define-callback/
+  - from: /guides/sign-into-web-app/aspnet/get-user-info/index.html
+    to: /docs/guides/sign-into-web-app/aspnet/get-user-info/
+  - from: /guides/sign-into-web-app/aspnet/next-steps/index.html
+    to: /docs/guides/sign-into-web-app/aspnet/next-steps/
+  - from: /guides/sign-into-web-app/aspnet/redirect-to-sign-in/index.html
+    to: /docs/guides/sign-into-web-app/aspnet/redirect-to-sign-in/
+  - from: /guides/sign-into-web-app/aspnet/require-authentication/index.html
+    to: /docs/guides/sign-into-web-app/aspnet/require-authentication/
+  - from: /guides/sign-into-web-app/springboot/before-you-begin/index.html
+    to: /docs/guides/sign-into-web-app/springboot/before-you-begin/
+  - from: /guides/sign-into-web-app/springboot/configure-packages/index.html
+    to: /docs/guides/sign-into-web-app/springboot/configure-packages/
+  - from: /guides/sign-into-web-app/springboot/create-okta-application/index.html
+    to: /docs/guides/sign-into-web-app/springboot/create-okta-application/
+  - from: /guides/sign-into-web-app/springboot/define-callback/index.html
+    to: /docs/guides/sign-into-web-app/springboot/define-callback/
+  - from: /guides/sign-into-web-app/springboot/get-user-info/index.html
+    to: /docs/guides/sign-into-web-app/springboot/get-user-info/
+  - from: /guides/sign-into-web-app/springboot/next-steps/index.html
+    to: /docs/guides/sign-into-web-app/springboot/next-steps/
+  - from: /guides/sign-into-web-app/springboot/redirect-to-sign-in/index.html
+    to: /docs/guides/sign-into-web-app/springboot/redirect-to-sign-in/
+  - from: /guides/sign-into-web-app/springboot/require-authentication/index.html
+    to: /docs/guides/sign-into-web-app/springboot/require-authentication/
```

Comment:
There are numerous existing redirects that need to be updated as well, see lines: 1241-1287 and 1502-1549 and elsewhere. (A lot of those original re-directs were incorrectly hard-coded as well; we might be able to reduce some lines). There's also a sprinkling of other lines, too. Do a search on "sign-into-web-app" in the conductor file.